src/index.js

```diff
@@ -1,11 +1,15 @@
 function Autosize(Alpine) {
     Alpine.directive('autosize', (el, {}, { cleanup }) => {
+        
+        if (el.hasAttribute('wire:model')) {
```

Comment:
```suggestion
        if (!el.hasAttribute('wire:ignore') && el.hasAttribute('wire:model')) {
```

Prevents an unnecessary DOM update (if `wire:ignore` is already set).